reload/fs.go

```diff
@@ -0,0 +1,60 @@
+package reload
```

Comment:
How about renaming it as something like "overlay", "live", "shadow", or... just simply "buff", something closer to what it does? Actually, it does not "reload" but act as a kind of "overlay". "reload" is fine but I would like to make it clear to understand when the user uses it.

Reply:
How about we move this functionality to the root of the gobuffalo/buffalo project? Then the name would be buffalo.FS which seems fitting to me?